codingTest_kjy/두정수사이의합.js

```diff
@@ -0,0 +1,11 @@
+function solution(a,b) {
+    let answer = 0;
+    if(a >= b)
+        for(let i=b; i <= a; i++){
+        answer = answer += i;
+}else{
+        for(let i=a; i <= b; i++)
+        answer = answer += i;
+}
+return answer;
+}
```

Comment:
```suggestion
function solution(a,b) {
    let answer = 0;
    if(a >= b)
        for(let i=b; i <= a; i++){
        answer = answer += i;
}else{
        for(let i=a; i <= b; i++)
        answer = answer += i;
}```suggestion
function solution(a,b) {
    let answer = 0;
    if(a >= b)
        for(let i = b; i <= a; i++){
        answer += i;
    }else{
        for(let j = a; j <= b; j++)
        answer += j;
    }
return answer;
}
```
코드 컨벤션 조금 손 봐드렸고, answer부분은 위처럼 줄일 수 있겠네요. 코멘트 확인하시면 어떤 흐름으로 코드 짜셨는지 답글 달아주세요!!
추가로 else 조건문에서 for반복문은 똑같이 i로 해도 되지만(저도 둘 다 i로 했습니다) 생각해보니까 좀 더 구분하기 쉽게 다른 문자로 해도 괜찮을 것 같네요. 보통은 i, j, k순으로 하고 필요에 따라서 p, q를 사용하기도 합니다.
return answer;
}
```

Reply:
확인했습니다!